receiver/azureeventhubreceiver/README.md

```diff
@@ -60,6 +72,18 @@ receivers:
     group: bar
     offset: "1234-5566"
     format: "azure"
+    # optional
+    time_format:
+      # All supported time format. Default is empty string array, which means using the current iso8601 parser. The format is based on https://pkg.go.dev/time#Layout. If no time-zone info, will use UTC time.
+      logs: ["01/02/2006 15:04:05","2006-01-02 15:04:05","2006-01-02T15:04:05Z07:00"]
+      metrics: [""]
```

Comment:
is this a valid option?

Reply:
updated. It's not a valid option and will cause parsing failure and fallback to iso8601.